lib/inMemFileDB.js

```diff
@@ -298,10 +298,12 @@ class InMemoryFileDB {
         }
 
         const clients = this.getClients();
+        let publishCount = 0;
 
         for (const i in clients) {
             if (clients.hasOwnProperty(i)) {
-                this.publishToClients(clients[i], type, id, obj);
+                // @ts-ignore
```

Comment:
Add `/** @returns {number} */` before `publishToClients` and remove `//@ts-ignore`